package.json

```diff
@@ -21,20 +22,28 @@
   },
   "homepage": "https://github.com/abahachuk/ptushki-backend#readme",
   "dependencies": {
+    "@types/morgan": "^1.7.35",
     "body-parser": "^1.18.3",
     "cross-env": "^5.2.0",
     "dotenv": "^7.0.0",
     "express": "^4.16.4",
     "express-session": "^1.15.6",
+    "express-validator": "^5.3.1",
+    "fs": "0.0.1-security",
+    "morgan": "^1.9.1",
+    "path": "^0.12.7",
```

Comment:
I wonder, what's the `path` module? How you use it?

Reply:
I think it is a mistake. Thanks.